docs/lang/articles/contribution/contributor_guide.md

```diff
@@ -131,143 +125,175 @@ python misc/code_format.py
-- CI is triggered each time you push a commit to an open PR.
-- A ✔️ on the left-hand side of a commit hash means that CI passes, whilst a ❌ means that CI fails.
+Following are some frequently used tags:
 
-### **PR reviewing & merging**
+- `[cuda]`: Backend-specific changes.
+- `[lang]`: Frontend language features, including syntax sugars.
+- `[ir]`: Intermediate representation-specific changes.
+- `[refactor]`: Code refactoring changes.
+- `[ci]`: CI/CD workflow-specific changes.
+- `[Doc]: Documentation updates.
 
-- Please try to follow these tips from Google:
-  - [Code Health: Understanding Code In](https://testing.googleblog.com/2018/05/code-health-understanding-code-in-review.html)
-  - [Code Health: Respectful Reviews == Useful](https://testing.googleblog.com/2019/11/code-health-respectful-reviews-useful.html)
+When introducing a new tag, ensure that you add it to [misc/prtags.json](https://github.com/taichi-dev/taichi/blob/master/misc/prtags.json) so that others can follow.
 
-- The master branch is required to have a **linear history**. https://idiv-biodiversity.github.io/git-knowledge-base/linear-vs-nonlinear.html
+:::
 
-- Ensure that the PR passes all **continuous integration (CI) tests**. See [Using continuous integration](./contributor_guide#using-continuous-integration).
+### PR review & merging checklist
 
-- Ensure that the title follows [PR tag rules](./contributor_guide#pr-title-format-and-tags).
+Folow this checklist during PR review or merging:
 
+1. Ensure that your PR title follows our [naming conventions](#pr-naming-conventions).
+2. Ensure that Taichi's master branch has a *linear history*. See the [Linear vs Non-Linear History](https://idiv-biodiversity.github.io/git-knowledge-base/linear-vs-nonlinear.html) for more information.
+3. Ensure that your PR passes all Continuous Integration (CI) tests before merging it.
+   CI is triggered each time you push a commit to an open PR. It builds and tests all commits in your PR in multiple environments. Keep an eye on the CI test results:
+   - A ✔️ on the left-hand side of a commit hash: CI has passed,
+   - A ❌ on the left-hand side of a commit hash: CI has failed.
 
+Here, we do not want to repeat some best practices summarized in the following Google blog articles. But do please spare a couple of minutes reading them if your PR is being reviewed or if you are reviewing a PR. They have our recommendation!
+  - [Code Health: Understanding Code In Review](https://testing.googleblog.com/2018/05/code-health-understanding-code-in-review.html)
+  - [Code Health: Respectful Reviews == Useful](https://testing.googleblog.com/2019/11/code-health-respectful-reviews-useful.html)
+  - [How to have your PR merged quickly](https://testing.googleblog.com/2017/06/code-health-too-many-comments-on-your.html)
 
 ## Frequently asked questions
 
 #### What if I didn't format my code style locally?
```

Comment:
This part seems duplicating https://github.com/Vissidarte-Herman/taichi/blob/d68a64521a625eb471245c7e94fd18ee09a18ed0/docs/lang/articles/contribution/contributor_guide.md?plain=1#L115-L120.

Reply:
Yes. It is a duplicate one. I'll remove it.